docker-compose.demo.yml

```diff
@@ -76,9 +76,6 @@ services:
         environment:
             # enable demo mode for UI ["true"/"false"]
             DEMO: "true"
-        labels:
-            - traefik.http.middlewares.sec-headers.headers.stsSeconds=0
-            - traefik.http.middlewares.sec-headers.headers.isDevelopment=true
```

Comment:
Although not critical, it would be nice to keep these to not have to clear the HSTS browser settings - I would guess this would mean an extra middleware in the `traefik.demo.yaml` and adding it to the gui router in there too...

Reply:
Will do, I just assumed they were already present in the file configuration and that I forgot to clear them previously.